pages/linux/mkvmerge.md

```diff
@@ -0,0 +1,16 @@
+# mkvmerge
+
+> Merge and extract multimedia streams
+> More information: <https://github.com/topics/mkvmerge>.
+
+- Print information about the source file:
+
+`mkvmerge --identify {{file.mkv}}`
```

Comment:
```suggestion
`mkvmerge --identify {{path/to/file}}`
```

Reply:
If only files in `mkv` format are allowed as input then it should be:
```suggestion
`mkvmerge --identify {{path/to/file.mkv}}`
```